mammoth/modules/embeddings.py

```diff
@@ -145,7 +147,25 @@ def __init__(
         # The embedding matrix look-up tables. The first look-up table
         # is for words. Subsequent ones are for features, if any exist.
         emb_params = zip(vocab_sizes, emb_dims, pad_indices)
-        embeddings = [nn.Embedding(vocab, dim, padding_idx=pad) for vocab, dim, pad in emb_params]
+
+        emb_params = zip(vocab_sizes, emb_dims, pad_indices)
+        if enable_embeddingless is False:
+            embeddings = [nn.Embedding(vocab, dim, padding_idx=pad) for vocab, dim, pad in emb_params]
+
+        else:
+            print("CREATING EMBEDDINGLESS")
```

Comment:
being pedantic here: would like to see a more informative log output (IFF not only for debugging purposes).
perhaps print using the logger.

Reply:
Fixed this. Was there for debugging purposes.